contrib/dockerd-rootless-setuptool.sh

```diff
@@ -212,6 +212,33 @@ init() {
 		fi
 	fi
 
+	# instruction: Ubuntu 24 enables kernel.apparmor_restrict_unprivileged_userns by default and needs an AppArmor profile
+	if [ ! -e "/etc/apparmor.d/usr.local.bin.rootlesskit" ] && [ -e "/etc/apparmor.d/abi/4.0" ] && [ -e "/proc/sys/kernel/apparmor_restrict_unprivileged_userns" ]; then
```

Comment:
This instruction is not needed for dpkg installation (/usr/bin/rootlesskit), and it is already printed by rootlesskit itself when needed